akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala

```diff
@@ -351,10 +353,7 @@ final class GraphInterpreter(
         def reportStageError(e: Throwable): Unit = {
           if (activeStage == null) throw e
           else {
-            // TODO: Get error reporting back
-            //val stage = stages(activeStage.stageId)
-
-            log.error(e, "Error in stage [{}]: {}", activeStage, e.getMessage)
+            log.error(e, "Error in stage [{}]: {}", activeStage.originalStage.getOrElse(activeStage), e.getMessage)
```

Comment:
Hm this won't show exception type though which is likely important, add `{}: {}` (e, ex.getMessage)?